### PR TITLE
Explain how to fetch the current tt_content UID

### PR DESCRIPTION
**Pull request by cweiske**
Resolves: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/3928

### CODE REVIEW

File: Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/CurrentContentObject.rst [hunk 1]
```diff
@@ -30,3 +30,4 @@ Example:
      * @var \TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer $currentContentObject
      */
     $currentContentObject = $request->getAttribute('currentContentObject');
+    $curTtContentUid = $currentContentObject->data['uid'];
```
Comment (brotkrueml) [inline Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/CurrentContentObject.rst:33]: Actually, I would avoid naming it `$curTtContentUid` - that is bad to read, instead `$currentContentUid` - there is no need to add `tt`.

**Comment by cweiske** [comment]
I've changed it but added a comment mentioning "tt_content", so that it's easier to find by developers searching for "tt_content uid"

File: Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/CurrentContentObject.rst [hunk 2]
```diff
@@ -30,3 +30,5 @@ Example:
      * @var \TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer $currentContentObject
      */
     $currentContentObject = $request->getAttribute('currentContentObject');
+    //ID of current tt_content record
```
Comment (brotkrueml) [inline Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/CurrentContentObject.rst:33]:
```suggestion

    // ID of current tt_content record
```

Comment (brotkrueml) [reply Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/CurrentContentObject.rst:33]: @cweiske Can you please commit the suggestion? In future PRs please allow us to change your adjustments. This makes it easier for us all.

**Comment by cweiske** [comment]
Done.

**Comment by cweiske** [comment]
I can't allow edits: 
2024-01-25 Auswahl_001

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Probably because I'm using the Mogic company account, not my personal account.

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**